apps.toml

```
@@ -2060,6 +2060,11 @@ level = 8
 state = "working"
 url = "https://github.com/YunoHost-Apps/mosquitto_ynh"
 
+[motioneye]
+category = "iot"
+state = "inprogress"
```

```suggestion
state = "working"
```

considering the only issue left seems to be the linter reporting the app as not in the catalog, let's set it directly as working ?